packages/venia-ui/lib/components/MiniCart/miniCart.css

```diff
@@ -43,9 +43,8 @@
     /* The header and footer heights may change based on PWA-116 and PWA-601. */
     grid-template-rows: 2rem 1fr minmax(2rem, auto);
     /* These heights may change based on PWA-605 and should be based on 1.5 and 2.5 visible items. */
-    height: 100%;
-    min-height: 31rem;
-    max-height: 45rem;
+    min-height: 10rem;
+    max-height: 32rem;
```

Comment:
The maximum looks right to me. Does the minimum have to be this low? When the component enters and exits the loading state, it shrinks and grows by quite a bit.

Reply:
I was going with the idea of `min-height=1 ITEM HEIGHT` and `max-height=2.5 ITEMS HEIGHT`. If the `min-height` is any bigger than that, there seems to be an empty space in case of 1 item in the cart. My CSS knowledge is kind of rusty, let me know if there is a cleaner solution than that.